modules/standard/Set.chpl

```diff
@@ -707,9 +719,12 @@ module Set {
     :return: A new set containing the symmetric difference of `a` and `b`.
     :rtype: `set(?t, ?)`
```

Comment:
Ditto parsafety and rtype.

Reply:
I just removed the `rtype` for all of these.